src/applications/simple-forms/21-10210/pages/claimantIdInfo.js

```diff
@@ -0,0 +1,28 @@
+export default {
+  uiSchema: {
+    claimantSSN: {
+      'ui:title': 'Social Security number',
+    },
+    vaFileNumber: {
+      'ui:title': 'VA file number (if applicable)',
+    },
+    vaInsuranceFileNumber: {
+      'ui:title': 'VA Insurance File Number (if applicable)',
+    },
+  },
+  schema: {
+    type: 'object',
+    required: ['claimantSSN'],
+    properties: {
+      claimantSSN: {
+        type: 'string',
```

Comment:
I know this is still a WIP, but just noticing no validation here.

Reply:
Fixed.